app/models/Db/Trun/Users.php

```diff
@@ -24,4 +24,9 @@ public function updateNameById($id, $name)
         $params     = array($name, $id);
         $connection->execute($query, $params);
     }
+
+    public function sideci_test()
```

Comment:
[code_sniffer] Public method name "Users::sideci_test" is not in camel caps format